hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+  - :>SetTxtone 0
+  - :>setchannel 1
+  - :>SetTxtone 1  
+  - :>SetTxtone 0
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command;
+- Setchannel 0 will select a frequency points to test, it can also select another frequency point to test by changing the channel number;
+- It can use SetPower [max] to set maximum output power;
+- The "SetTxTone 1" command sets the radio to CW transmit mode, and the radio will transmit CW signal. 
+- The CW signal transmission can be disabled by "SetTXtone 0" command. Then the Setchannel command can be issued in idle radio state.
+
+## 4.2. The Signal Analyser setup and test example
+It is unlike CTune calibration, the fundamental and harmonics test requires big frequency span and big RBW for fast sweep measurement.
+- Set appropriate reference level for the test, e.g. if your maximum output power is +20dBm, you can set reference level to +23dBm which is greater than maximum power by 2-3 dB;
+- Set the frequency span to contain the fundamental and desired harmonics for the test. Usually 5 times fundamental frequency is desired. But according to the regulation, the maximum harmonic is tested at 10 times of fundament;
+- Set the RBW as big as possible, as long as the noise floor of SA should below the limit by 3 dB to 10dB.
+- Set the VBW to 3 times of RBW;
+- Set the trace mode to Max Hold;
+- Set the detector mode to Peak detector;
+- It can set the cable loss in fundamental and harmonics frequency points, this will count the cable loss in the display results;
+- It can set a marker table to display the signal strenghth at fundamental and harmonics.
+
+The following picture is a test example:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\Tx-Power-and-Harmonics.png">  
+</div>
+
+
+# 5. Conducted Rx sensitivity measurement
+Sensitivity is basic test item for Rx performance. Usually it use a Signal Generator(SG) to generate a modulated signal and inject it into radio chip. Then the radio chip can demodulate the signal into data, the Railtest tool can use the data to count the BER. Once the BER critia is achieved, then the SG power level is called sensitivity.
+
+## 5.1. The Signal Generator settings
+At beginning of the test, it should configure SA to generate right modulated signal. Following steps will configure a GFSK modulated signal:
+
+- Set the SG to Custom Mode;
+- Set IQ data to PN9;
+- Set modulation type to 2-level FSK;
+- Set filter to 0.5Bt Gaussian type;
+- Set symble rate to the desired data rate, symble rate=data rate for 2GFSK modulation;
+- Set frequency deviation to desired frequency deviation; 
+- Set frequency to desired operating frequency;
+- RF: ON
+- MOD: ON
+- Tune signal strength level;
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\BER3.png">  
+</div>
+
+## 5.2. RailTest commands and test result
+The RailTest commands are listed below for Rx BER test: 
+  - :>rx 0
+  - :>Setchannel 0 
+  - :>SetBerconfig 1000
+  - :>BerRx 1
+  - :>......  
```

Comment:
what is the intent of this line?  Is the :> required in each of the bullets?

Reply:
Modified.